README.md

```diff
@@ -1,2 +1,454 @@
+
+#### Examples
+
+``` json
+"fct": [
+  {
+    "challenge": "abcdef",
+    "from": "example.com"
+  },
+  {
+    "sha3_256": "B94D27B9934D3E08A52E52D7DA7DABFAC484EFE37A5380EE9088F7ACE2EFCDE9",
+    "msg": "hello world"
+  }
+]
+```
+
+### 3.2.5 Attenuation Scope
+
+The attenuation scope (i.e. UCAN output, or "caveats") MUST be an array of heterogeneous access scopes (defined below). This array MAY be empty.
+
+This array MUST contain some or none of the following: a strict subset (attenuation) of the proofs, resources originated by the `iss` DID (i.e. by parenthood), and resources that compositions of others (see rights amplification). This scoping also includes time ranges, making the witnesses that start latest and end soonest the lower and upper time bounds.
+
+The attenuation field MUST contain either a wildcard (`*`), or an array of JSON objects. A JSON capability MUST contain the `with` and `can` field, and MAY contain additional fields needed to describe the capability.
+
+``` json
+{
+  "with": $RESOURCE,
+  "can": $ABILITY
+}
+```
+
+#### 3.2.4.1 Resource Pointer
+
+A resource describes the noun of a capability. The resource pointer MUST be provided in [URI](https://datatracker.ietf.org/doc/html/rfc3986) format. Arbitrary and custom URIs MAY be used, provided that the intended recipient is able to decode the URI. The URI is merely a unique identifier to describe the pointer to -- and within -- a resource.
+
+The same resource MAY be addressed with several URI formats. For instance a database may be addressed at the level of direct memory with `file`, via `sqldb` to gain access to SQL semantics, `http` to use web addressing, and `dnslink` to use Merkle DAGs inside DNS `TXT` records. 
+
+Resource pointers MAY also include wildcards (`*`) to indicate "any resource of this type" -- even if not yet created -- bounded by attenuation witnesses. These are generally used for account linking. Wildcards are not required to delegate longer paths, as paths are generally taken as OR filters.
+
+| URI | Meaning |
+|-------|---------|
+| `{"with": "mailto:boris@fission.codes", ...}`      | A single email address        |
+| `{"with": "my:*", ...}` | All resources that the iss has access to, including future ones |
+| `{"with": "my:dnslink", ...}` | All DNSLinks that the iss has access to, including future ones |
+| `{"with": "dnslink://myapp.fission.app", ...}` | A mutable pointer to some data |
+
+#### 3.2.4.2 Ability
+
+The `can` field describes the verb portion of the capability: an action, potency, or ability. For instance, the standard HTTP methods such as `GET`, `PUT`, and `POST` would be possible `can` values for an `http` resource. Arbitrary semantics can be described, but must be a valid way to describe actions on the resource.
```

Comment:
Ah, this PR doesn't reflect the latest state of the spec. We've merged a couple PRs since. We've consolidated on "ability" everywhere, and simply removed the other terms from the document.